redwood-gradle-plugin/build.gradle

```diff
@@ -47,12 +47,6 @@ dependencies {
 
 gradlePlugin {
   plugins {
-    redwood {
-      id = "app.cash.redwood"
-      displayName = "Redwood"
-      description = "Redwood Compose Gradle plugin"
-      implementationClass = "app.cash.redwood.gradle.RedwoodComposePlugin"
-    }
```

Comment:
Like our build's `composeCompiler()` helper, this plugin served only to add the Compose compiler to a module. Use `org.jetbrains.kotlin.plugin.compose` instead!

(Will be put in change log)